ranger2/aws/ranger_asg/ranger_bootstrap.sh

```diff
@@ -0,0 +1,4 @@
+source /usr/lib/qubole/bootstrap-functions/hive/ranger-client.sh
+
+#Usage for any Hive Cluster: install_ranger -h "RangerDNS/LB" -p Port -S "SolrDNS/LB" -P port
+install_ranger -h xxxx-ranger-alb-xxxxxx.us-east-1.elb.amazonaws.com -p 80 -S xxx-solr-alb-xxxx.us-east-1.elb.amazonaws.com -P 80
```

Comment:
should we be hardcoding us-east-1 here? I think the region should be configurable(not even take from the session credentials)

Reply:
Fixed added comments